json_examples/task-min_eeg.json

```diff
@@ -0,0 +1,9 @@
+{
+  "TaskName":"Fancy stuff",
+  "PowerLineFrequency":50,
```

Comment:
empty

Reply:
+1 for dora's comment: please leave TaskName empty